dht_test.go

```diff
@@ -22,16 +24,16 @@ import (
 	opts "github.com/libp2p/go-libp2p-kad-dht/opts"
 	pb "github.com/libp2p/go-libp2p-kad-dht/pb"
 
-	cid "github.com/ipfs/go-cid"
+	"github.com/ipfs/go-cid"
```

Comment:
let's restore these aliases. makes it easier for people to track down

Reply:
We generally order imports this way:

```go
import (
    [stdlib...]

    [libp2p...] (without aliases)

    [multiformats...]

    [everything else in groups, e.g. all ipfs imports together...]
)
```